pyproject.toml

```
@@ -126,6 +126,7 @@ include = ["exspy*"]
   "*hspy",
   "*.yaml",
   "*.msa",
+  "data/elements/*.json",
```

Interestingly, if this was doing working as expected, it would break the packaging because it should be:
```suggestion
  "exspy/_misc/*.json",
```
Not sure if this is a bug somewhere or something is setup incorrectly in `pyproject.toml`!
Checking the package and test CI workflow, the `json` files are included in the tarball and the wheel.